hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/SCMNodeManager.java

```diff
@@ -1265,6 +1275,38 @@ private void nodeUsageStatistics(Map<String, String> nodeStatics) {
     nodeStatics.put(UsageStatics.STDEV.getLabel(), decimalFormat.format(dev));
   }
 
+  private void nodeSpaceStatistics(Map<String, String> nodeStatics) {
+    if (nodeStateManager.getAllNodes().size() < 1) {
+      return;
+    }
+    long capacityByte = 0;
+    long scmUsedByte = 0;
+    long remainingByte = 0;
+    for (DatanodeInfo dni : nodeStateManager.getAllNodes()) {
+      List<StorageReportProto> storageReports = dni.getStorageReports();
+      if (storageReports != null && !storageReports.isEmpty()) {
+        for (StorageReportProto storageReport : storageReports) {
+          capacityByte += storageReport.getCapacity();
+          scmUsedByte += storageReport.getScmUsed();
+          remainingByte += storageReport.getRemaining();
```

Comment:
There are two more relevant `StorageReportProto`, `committed` and `freeSpaceToSpare`. Do we need to include these?

Reply:
Thanks @ivandika3  for the comment and review.
You mean we only need to keep Capacity and Scmused here, right?

Reply:
I meant that `StorageReportProto` contains two possible relevant fields `committed` and `freeSpaceToSpare`, but here we only get the `capacity`, `scmUsed`, and `remaining`. Do we need to include the `committed` and `freeSpaceToSpare` here? I'm fine if you don't think it's relevant for now.

Reply:
I think committed and freeSpaceToSpare are secondary and don't need to be shown here. Therefore, I didn't handle them.

Reply:
Ok, can be added in the future.